### DISCUSSION TITLE
Alertmanager data source page should mention how to configure prometheus implementation

### DISCUSSION THREAD

**Discussion by johnymachine** [grafana/grafana#57191] [Data Sources]
After extensive search I found the answer here: https://community.grafana.com/t/adding-prometheus-alertmanager-in-grafana/69790/4

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
